spec/client/viewing/RenderableViewSpec.js

```diff
@@ -30,11 +31,12 @@ describe("RenderableView", function() {
             });
 
             it("combines the data in the model and returned from logic", function() {
-                expect(view.templateData()).toEqual({
+                expect(view.templateData()).toEqual(jasmine.objectContaining({
```

Comment:
since the object also contains .views now, it won't ever be equal to only the logic and model data combined